tools/commands/check_tidy.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Do you have any plans to backport it to original place? 😄

Reply:
I don't get you... could you tell me more detail?